modules/ROOT/pages/dw-binaries-functions-fromhex.adoc

```diff
@@ -22,14 +22,15 @@ Transforms a hexadecimal string into a binary.
 === Example
 
 This example transforms a hexadecimal string to "Mule".
+To show you the type, it outputs data in the `application/dw` format. 
```

Comment:
I'm not sure of what you mean by "To show you the type".
Do you mean something like "To make the resulting type clear, the example displays the output in `application/dw` format."?